javascript/javascript1/week3/nonoyes.js

```diff
@@ -0,0 +1,57 @@
+// Save a note
+
+const notes = [];
+function saveNote(content, id) {
+  notes.push({ content, id });
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+// Get a note
+
+function getNote(id) {
+  for (let note of notes) {
+    if (note.id == isNaN(id)) {
+      return "Error";
```

Comment:
`isNaN` returns true or false, I'm not sure what you are trying to do here, but if you try to check if it's a number you need to change your condition, if you are trying to convert it to something else then you need to use coercion.